src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -36,7 +36,7 @@ def resolve_resource_group(cmd, config_store_name):
     config_store_client = cf_configstore(cmd.cli_ctx)
     all_stores = config_store_client.list()
     for store in all_stores:
-        if store.name == config_store_name:
+        if store.name.lower() == config_store_name.lower():
             # Id has a fixed structure /subscriptions/subscriptionName/resourceGroups/groupName/providers/providerName/configurationStores/storeName"
```

Comment:
Store name is case sensitive. Although we use it as part of the DNS record, so we don't allow store name "ABC" and "abc" both to exist.